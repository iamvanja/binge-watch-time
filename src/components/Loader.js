import React from 'react'

const Loader = () => (
  <div className='loader'>
    <span>&bull;</span>
    <span>&bull;</span>
    <span>&bull;</span>
  </div>
)

export default Loader
