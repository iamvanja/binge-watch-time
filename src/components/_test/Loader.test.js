import React from 'react'
import { shallow } from 'enzyme'
import Loader from '../Loader'

describe('Loader', () => {
  it('renders Loader', () => {
    const wrapper = shallow(
      <Loader />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
