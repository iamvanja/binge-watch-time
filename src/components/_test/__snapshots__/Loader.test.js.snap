// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders Loader 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span>
          •
</span>,
        <span>
          •
</span>,
        <span>
          •
</span>,
      ],
      "className": "loader",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "•",
        },
        "ref": null,
        "rendered": "•",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "•",
        },
        "ref": null,
        "rendered": "•",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "•",
        },
        "ref": null,
        "rendered": "•",
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span>
            •
</span>,
          <span>
            •
</span>,
          <span>
            •
</span>,
        ],
        "className": "loader",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "•",
          },
          "ref": null,
          "rendered": "•",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "•",
          },
          "ref": null,
          "rendered": "•",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "•",
          },
          "ref": null,
          "rendered": "•",
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
